docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -277,16 +277,45 @@ in the partition::
     DELETE FROM iceberg.testdb.customer_orders
     WHERE country = 'US' AND customer = 'Freds Foods'
 
-Rolling back to a previous snapshot
------------------------------------
+
+
+Snapshots
+---------
 
 Iceberg supports a "snapshot" model of data, where table snapshots are
 identified by an snapshot IDs.
 
 The connector provides a system snapshots table for each Iceberg table.  Snapshots are
```

Comment:
pre-existing: "system snapshots table" reads a bit weird. May be rephrase to "a system table containing snapshot information for every iceberg table"